cvat-core/src/annotation-format.js

```diff
@@ -127,108 +105,41 @@
         * @memberof module:API.cvat.classes
         * @hideconstructor
     */
-    class AnnotationFormat {
+    class AnnotationFormats {
```

Comment:
Maybe also rename this file?

Reply:
If it makes sense to the code style, I have no preferences here.

Reply:
It's just suggestion. I also do not have preferences here

Reply:
Done